games-board/cockatrice/cockatrice-20161030.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils
+
+DESCRIPTION="An open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+SRC_URI="${HOMEPAGE}/archive/2016-10-30-Release.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="server +client +oracle"
+
+DEPEND="
+	dev-libs/libgcrypt:0
+	dev-libs/protobuf
+	dev-qt/linguist-tools:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtprintsupport:5
+	client? (
+		dev-qt/qtgui:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtsvg:5
+	)"
+RDEPEND="
+	dev-qt/qtcore:5 "
+
+# As the default help/about display the sha1 we need it
+SHA1='3498b16'
+
+S="${WORKDIR}/Cockatrice-2016-10-30-Release"
+
+src_configure() {
+	local mycmakeargs=(
+		-DWITH_CLIENT=$(usex client 1 0)
+		-DWITH_ORACLE=$(usex oracle 1 0)
+		-DWITH_SERVER=$(usex server 1 0)
+		-DICONDIR="/usr/share/icons"
+		-DDESKTOPDIR="/usr/share/applications" )
+
+	# Add date in the help about, come from git originally
+	sed -i 's/^set(PROJECT_VERSION_FRIENDLY.*/set(PROJECT_VERSION_FRIENDLY \"'${SHA1}'\")/' cmake/getversion.cmake || die "Sed failed!"
```

Comment:
Yes the value added is to have the sha displayed in the help/about menu (instead of nothing),
see https://github.com/Cockatrice/Cockatrice/issues/2268